### PR TITLE
Feature list structure

### PR DESCRIPTION
**Pull request by marilirulita**
This project is built a to-do list adding and managing DOM elements using javascript and Webpack

- First version creates an array of objects which represent the item list
- Create a constructor of list object which represents each element of the tasks list.
- Add a button to future delete elements in the list that are completed.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello  @marilirulita 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do List</title>
+</head>
+<body>
+    <h1>To Do List</h1>
+    <div id="container">
+        <h3>Today´s To Do</h3>
+        <input type="text" name="task" id="new-task" placeholder="Add to your list..." />
+        <div id="list-elem">
```
Comment (mikenath223) [inline src/index.html]: - [x] Kindly make the project match the given design:

| Original | Yours |
| ----- | ------ |
|  |  |

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+// import _ from 'lodash';
```
Comment (mikenath223) [inline src/index.js]: - [x] Please remove the commented code.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+import './style.css';
+
+const tasksList = [];
+let ids = 0;
+
+function Task(description, completed, index) {
+  this.description = description;
+  this.completed = completed;
+  this.index = index;
+}
```
Comment (mikenath223) [inline src/index.js]: - [x] Kindly change these and other regular functions to es6 arrow function syntax following the JavaScript best practices guide.

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+// tasksList.push(newTask);
+// }
+
+// const taskInput = document.getElementById("new-task");
+
+// taskInput.addEventListener("keyup", function(event) {
+//  if (event.characterCode == 13) {
+//    add();
+//  }
+// });
```
Comment (mikenath223) [inline src/index.js]: - [x] Please remove this commented code.

**Review by GzimAsani — APPROVED** [review]
# Hello @marilirulita     

## Thank you for implementing all the requirements asked previously. Your project is **ACCEPTED :green_circle:** and I wish you the best of luck in your next projects :+1: 

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
 You can also consider:
 
- [  **_Optional_** ] Your delete button is not being appended to the page. Kindly add that.

 Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**